docker/web-console/standalone/Dockerfile

```diff
@@ -15,62 +15,76 @@
 # limitations under the License.
 #
 
+
+#~~~~~~~~~~~~~~~~~~#
+#  Frontend build  #
+#~~~~~~~~~~~~~~~~~~#
 FROM node:8-slim as frontend-build
 
 ENV NPM_CONFIG_LOGLEVEL error
 
 WORKDIR /opt/web-console
 
-# Install node modules for frontend.
-COPY frontend/package*.json frontend/
-RUN (cd frontend && npm install --no-optional)
-
-# Copy source.
+# Install node modules and build sources
 COPY frontend frontend
+RUN cd frontend && \
+    npm install --no-optional && \
+    npm run build
 
-RUN (cd frontend && npm run build)
 
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+#  Web Console Standalone assemble  #
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
 FROM node:8-slim
 
 ENV NPM_CONFIG_LOGLEVEL error
 
-# Update package list & install.
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 0C49F3730359A14518585931BC711F9BA15703C6 \
-    && echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.4 main" | tee /etc/apt/sources.list.d/mongodb-org-3.4.list
-
-# Update package list & install.
-RUN apt-get update \
-    && apt-get install -y nginx-light mongodb-org-server dos2unix \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# Install global node packages.
+# Install global node packages
 RUN npm install -g pm2
 
-WORKDIR /opt/web-console
-
-COPY docker/standalone/docker-entrypoint.sh docker-entrypoint.sh
-
-RUN chmod +x docker-entrypoint.sh \
-    && dos2unix docker-entrypoint.sh
+# Update software sources and install missing applications
```

Comment:
May be moving this code block is not good idea (this hide real changes and increase diff)?

Reply:
I did not quite understand what was comment about, but I've merged 2 RUN commands into 1 for less layers number and better control over software installation process.